docs/backend/behaviors.md

```diff
@@ -11,12 +11,320 @@ myst:
+| plone.tableofcontents  | Table of contents | Adds a table of contents |
+| plone.thumb_icon | Thumbs and icon handling | Options to suppress thumbs and/or icons and to override thumb   size in listings, tables etc.
+| plone.versioning  | Versioning | Versioning support with CMFEditions |
+| volto.blocks  | Blocks | Enables Volto Blocks support |
+| volto.blocks.editable.layout  | Blocks (Editable Layout) | Enables Volto Blocks (editable layout) support |
+| volto.head_title  | Head title field | Adds Head title field |
+| volto.navtitle  | Navigation title | Navigation title used in sections, menus and doormats |
+| volto.preview_image  | Preview Image | Preview image for listings |
+| volto.preview_image_link | Preview Image Link | Preview image for listings based on links |
+
+## Adding or removing a behavior from a content type
+
+There are two ways to add or remove a behavior on a content type:
+
+- Through the web using the {guilabel}`Content Types` control panel.
+- Using a custom add-on GenericSetup profile.
+
+### Through the Web
+
+1. Go to the {guilabel}`Site Setup` and chose the {guilabel}`Content Types` control panel.
+2. Select the content type to which you want to add or remove a behavior.
+3. Then click on the {guilabel}`Behaviors` tab of the settings of the content type.
+4. A list of all available behaviors appears.
+   Select or deselect the checkbox of the behavior you want to add to or remove from the type.
+5. Save the form by clicking on the {guilabel}`Save` button at the bottom of the page.
+
+### Using a GenericSetup profile
+
+Given you already have a custom add-on with a `profiles/default` directory, and you created a custom behavior named `mybehavior.subtitle`.
+
+If you want to enable a behavior on an existing content type, create a new directory `types` under `profiles/default`.
+In the `types` directory, create a file named the same as the content type you want to change.
+In the example here, you want to add a behavior to the built-in Event content type.
+The file to create is named `Event.xml` and it is a {term}`Factory Type Information` (FTI) definition.
+You need to change only the behaviors configuration.
+All other parts can be ignored.
+This looks like so:
+
+```xml
+<?xml version="1.0"?>
+<object
+    i18n:domain="plone"
+    meta_type="Dexterity FTI"
+    name="Event"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+  <property name="behaviors" purge="false">
+    <element value="myproject.subtitle" />
+  </property>
+</object>
+```
+
+After you apply the profile (or uninstall and install the custom add-on), the behavior is available on the Event content type.
+
+
+## Custom behaviors
+
+There are two types of behaviors:
+
+Schema-only behaviors
+: These behaviors have only a schema with fields.
+
+Full behaviors
+: A python class containing the logic of the behavior, an interface or schema defining the contract of the behavior, and a marker interface applied to the content type.
```

Comment:
```suggestion
: A python class containing the logic of the behavior, an interface or schema defining the contract of the behavior, and a marker interface applicable to a content type.
```

I would say the behavior **can** be applied to a content type of the developers choice.